cfgov/v1/tests/templatetags/test_email_popup.py

```diff
@@ -27,32 +26,32 @@ def test_no_popups_configured_returns_blank(self):
         self.assertEqual(response, '')
 
     @override_settings(
-        EMAIL_POPUP_URLS={'foo': ['/page/configured/']},
+        EMAIL_POPUP_URLS={'foo': {'en': ['/page/configured/']}},
         FLAGS={}
     )
     def test_popup_configured_but_no_flag(self):
         response = self.render('/page/configured/')
         self.assertEqual(response, '')
 
     @override_settings(
-        EMAIL_POPUP_URLS={'foo': ['/page/configured/']},
-        FLAGS={'EMAIL_POPUP_FOO': {'boolean': False}} 
+        EMAIL_POPUP_URLS={'foo': {'en': ['/page/configured/']}},
+        FLAGS={'EMAIL_POPUP_FOO': {'boolean': False}}
     )
     def test_popup_configured_but_flag_false(self):
         response = self.render('/page/configured/')
         self.assertEqual(response, '')
 
     @override_settings(
-        EMAIL_POPUP_URLS={'foo': ['/page/configured/']},
-        FLAGS={'EMAIL_POPUP_FOO': {'boolean': False}} 
+        EMAIL_POPUP_URLS={'foo': {'en': ['/page/configured/']}},
+        FLAGS={'EMAIL_POPUP_FOO': {'boolean': False}}
     )
     def test_popup_configured_and_flag_true_but_path_doesnt_match(self):
         response = self.render('/page/not/configured/')
         self.assertEqual(response, '')
 
     @override_settings(
-        EMAIL_POPUP_URLS={'oah': ['/page/configured/']},
-        FLAGS={'EMAIL_POPUP_OAH': {'boolean': True}} 
+        EMAIL_POPUP_URLS={'oah': {'en': ['/page/configured/']}},
+        FLAGS={'EMAIL_POPUP_OAH': {'boolean': True}}
     )
     def test_popup_configured_and_flag_true_and_path_matches(self):
```

Comment:
It'd be nice to add an additional unit test like this one that confirms that the language attribute `lang="something"` makes it into the template.

Reply:
Sure, what would a test like this look like?

Reply:
I'm no longer convinced an extra test is needed now that you've removed the language configuration from the settings dictionary. If you were using the dictionary (as before 876592d), it'd be nice to confirm that the dict key was being used and getting into the template. As it is now, the test would just be verifying that template content in one file (like [here](https://github.com/cfpb/cfgov-refresh/pull/3922/files#diff-afca900319cd83827ac48ee8eacf7498R3)) shows up when being rendered in another file ([here](https://github.com/cfpb/cfgov-refresh/pull/3922/files#diff-fafa50303a4ed1aaa70f935f5e4e29d1R15)). I don't think that adds much value.